### PR TITLE
Refactor:  Testimonial

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
 - Trello task - https://trello.com/c/mSbMJ7X5/144-uag-block-wise-cleanup

### Types of changes
Breaking change

### How has this been tested?
 - Check Trello Card.

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/testimonial/attributes.js [hunk 1]
```diff
@@ -2,350 +2,349 @@
...
-		type: "number",
-		default: 50
+	backgroundOpacity: {
+		type: 'number',
+		default: 50,
 	},
-	arrowColor:{
-		type: "string",
-		default: "#333"
+	arrowColor: {
```
Comment (rohitp-bsf) [inline src/blocks/testimonial/attributes.js:340]: Duplicate `arrowColor` key!

File: src/blocks/testimonial/inline-styles.js [hunk 2]
```diff
@@ -58,176 +58,237 @@ function TestimonialStyle( props ) {
...
 
-	styling_css = generateCSS( selectors, id )
-	styling_css += generateCSS( tablet_selectors, `${id}.uagb-editor-preview-mode-tablet`, true, "tablet" )
-	styling_css += generateCSS( mobile_selectors, `${id}.uagb-editor-preview-mode-mobile`, true, "mobile" )
+	let styling_css = '';
+	const id = `.uagb-block-${ props.clientId.substr( 0, 8 ) }`;
 
-	return styling_css
+	styling_css = generateCSS( selectors, id );
+	styling_css += generateCSS(
```
Comment (rohitp-bsf) [inline src/blocks/testimonial/inline-styles.js:278]: Camel caseing!

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]

File: src/blocks/testimonial/attributes.js (hunk 1 above)
Comment (supriyajjadhav) [reply src/blocks/testimonial/attributes.js]: Removed

File: src/blocks/testimonial/inline-styles.js (hunk 2 above)
Comment (supriyajjadhav) [reply src/blocks/testimonial/inline-styles.js]: Updated variable names

File: src/blocks/testimonial/settings.js [hunk 3]
```diff
@@ -0,0 +1,1602 @@
+import times from 'lodash/times';
```
Comment (rohitp-bsf) [inline src/blocks/testimonial/settings.js:1]: Remove this dependency and use alternative!

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]

File: src/blocks/testimonial/settings.js (hunk 3 above)
Comment (supriyajjadhav) [reply src/blocks/testimonial/settings.js]: Removed

**Review by RajkiranBagal — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**